src/templates/organization.js

```diff
@@ -183,6 +183,7 @@ export const query = graphql`
         }
         LinkedIn_Profiles {
           data {
+            About
```

Comment:
@jasonLaster if we could just remove this (it goes to the client) this is very ready to merge!